curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/create-a-set-of-radio-buttons.english.md

```diff
@@ -30,7 +30,7 @@ tests:
   - text: Your page should have two radio button elements.
     testString: assert($('input[type="radio"]').length > 1, 'Your page should have two radio button elements.');
   - text: Give your radio buttons the <code>name</code> attribute of <code>indoor-outdoor</code>.
-    testString: assert($('label > input[type="radio"]').filter("[name='indoor-outdoor']").length > 1, 'Give your radio buttons the <code>name</code> attribute of <code>indoor-outdoor</code>.');
+    testString: assert($('input[type="radio"]').filter("[name='indoor-outdoor']").length > 1, 'Give your radio buttons the <code>name</code> attribute of <code>indoor-outdoor</code>.');
```

Comment:
```suggestion
    testString: assert($('label ~ input[type="radio"]').filter("[name='indoor-outdoor']").length > 1, 'Give your radio buttons the <code>name</code> attribute of <code>indoor-outdoor</code>.');
```

Reply:
The challenge is asking the camper to _nest_ the inputs inside labels. Using the general sibling combinator is going to make the test fail.

The point of removing the child selector from the name attribute test is just so we don't confuse the camper by triggering the assertion for the name attribute just because they forgot to nest the input.

The check for the nesting is handle in the third assertion test.

Reply:
Ah, I see